src/foraging_gui/Foraging.py

```diff
@@ -2987,7 +2987,9 @@ def _Open(self,open_last = False,input_file = ''):
                 # list of mice, experimenters, and two week in chronological order form date modified
                 mice, experimenters, two_week = self._Open_getListOfMice()
                 # only add mice from two weeks in drop down.
-                W = MouseSelectorDialog(self, [m + ' ' + n for m, n in zip(two_week, experimenters)])
+                print(mice)
```

Comment:
Why print(mice) and print(two_week) here?

Reply:
I need to do one more test in 446 but integration testing is going on. I will take out before merging!